src/main/java/org/junitpioneer/internal/PioneerAnnotationUtils.java

```diff
@@ -250,6 +259,16 @@ private static <A extends Annotation> List<A> findOnType(Class<?> element, Class
 				.collect(Collectors.toList());
 	}
 
+	/**
+	 * A helper utility method for {@link org.junitpioneer.jupiter.cartesian.CartesianTest} for finding
+	 * argument sources on parameters.
+	 *
+	 * @param testMethod the test method for which we want to find parameter arguments sources
+	 * @return a list of found arguments source annotations, including {@link CartesianArgumentsSource} annotations
```

Comment:
I like to highlight if the `List` can be null or will at least return an not null, but empty list (e.g. via `Collections.emptyList()`

Comment fits many places, changed in this PR.